src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -43,137 +49,141 @@ const defer = function defer() {
+    this.scene.layersConfiguration.setLayerAttribute(layer.id, 'sequence', 0);
 
-};
+    this.viewerDiv.dispatchEvent(eventLayerAdded);
 
-ApiGlobe.prototype.getProtocolProvider = function getProtocolProvider(protocol) {
-    return this.scene.scheduler.getProtocolProvider(protocol);
+    return layer;
 };
 
 /**
- * This function gives a chance to the matching provider to pre-process some
- * values for a layer.
+ * This function adds an imagery layer to the scene using a JSON file. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
+ * @constructor
+ * @param {Layer} layer.
+ * @return     {layer}  The Layer.
  */
-function preprocessLayer(layer, provider) {
-    if (!layer.updateStrategy) {
-        layer.updateStrategy = {
-            type: STRATEGY_MIN_NETWORK_TRAFFIC,
-        };
-    }
 
-    if (provider.tileInsideLimit) {
-        layer.tileInsideLimit = provider.tileInsideLimit.bind(provider);
-    }
-
-    if (provider.tileTextureCount) {
-        layer.tileTextureCount = provider.tileTextureCount.bind(provider);
-    }
-
-    if (provider.preprocessDataLayer) {
-        provider.preprocessDataLayer(layer);
-    }
-}
-
-/**
- * Init the geometry layer of the Scene.
- */
-ApiGlobe.prototype.init = function init() {
-    const map = this.scene.getMap();
-    map.tiles.init(map.layersConfiguration.getGeometryLayers()[0]);
+ApiGlobe.prototype.addImageryLayerFromJSON = function addImageryLayerFromJSON(url, parentLayerId) {
+    return Fetcher.json(url).then(result => this.addImageryLayer(result, parentLayerId));
 };
 
 /**
- * Add the geometry layer to the scene.
+ * This function adds an imagery layer to the scene using an array of JSON files. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
+ * @constructor
+ * @param {Layers} array - An array of JSON files.
+ * @return     {layer}  The Layers.
  */
-ApiGlobe.prototype.addGeometryLayer = function addGeometryLayer(layer) {
-    preprocessLayer(layer, this.scene.scheduler.getProtocolProvider(layer.protocol));
-    const map = this.scene.getMap();
-    map.layersConfiguration.addGeometryLayer(layer);
+ApiGlobe.prototype.addImageryLayersFromJSONArray = function addImageryLayersFromJSONArray(urls, parentLayerId) {
+    const proms = [];
+
+    for (const url of urls) {
+        proms.push(Fetcher.json(url).then(layer => this.addImageryLayer(layer, parentLayerId)));
+    }
+
+    return Promise.all(proms);
 };
 
 /**
- * This function adds an imagery layer to the scene. The layer id must be unique. The protocol rules wich parameters are then needed for the function.
+ * Add an elevation layer to the map. Elevations layers are used to build the terrain.
+ * Only one elevation layer is used, so if multiple layers cover the same area, the one
+ * with best resolution is used (or the first one is resolution are identical).
```

Comment:
These typo are already in master so I won't fix them in this PR (even if it would be a good idea to fix them at some point).